package.json

```diff
@@ -54,10 +56,6 @@
     "@rollup/plugin-alias": "^3.1.1",
     "@rollup/plugin-node-resolve": "^9.0.0",
     "@rollup/plugin-typescript": "^5.0.2",
-    "@storybook/addon-actions": "^6.0.28",
-    "@storybook/addon-essentials": "^6.0.28",
-    "@storybook/addon-links": "^6.0.28",
-    "@storybook/web-components": "^6.0.28",
```

Comment:
You don't need any of the Storybook packages when using `@open-wc/demoing-storybook`